packages/builder/src/pages/builder/workspace/[application]/data/table/[tableId]/_components/ViewNavBar.svelte

```diff
@@ -237,9 +238,13 @@
     class:active={tableId === activeId}
     on:contextmenu={openTableContextMenu}
   >
-    <div class="nav-item__title">
-      {table?._id === TableNames.USERS ? "App users" : table?.name || ""}
-    </div>
+    <AbsTooltip
+      text={table?._id === TableNames.USERS ? "App users" : table?.name || ""}
```

Comment:
Put this table name resolution into a reactive variable at the top of the file. Nice to keep the it in one place 👍